support/cas-server-support-oauth-api/src/main/java/org/apereo/cas/ticket/refreshtoken/OAuth20RefreshTokenFactory.java

```diff
@@ -24,12 +24,14 @@
      * @param ticketGrantingTicket the ticket granting ticket
      * @param scopes               the scopes
      * @param clientId             the client id
+     * @param descendantTicket     the descendant access tokens
      * @param requestClaims        the request claims
      * @return the refresh token
      */
     OAuth20RefreshToken create(Service service, Authentication authentication,
                                TicketGrantingTicket ticketGrantingTicket,
                                Collection<String> scopes,
                                String clientId,
+                               String descendantTicket,
```

Comment:
Looks like this might be better served as either `String accessToken`, or a collection of `Collection<String> accessTokens`

Reply:
Done